package.json

```diff
@@ -28,7 +28,7 @@
     "@babel/types": "^7.0.0",
     "ansi-to-html": "^0.6.4",
     "babel-code-frame": "^6.26.0",
-    "babel-core": "^6.25.0",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
This is basically `@babel/core` 7 right? So this contains breaking AST and types changes? That's kind of what we wanna prevent by converting ASTs or re-parsing the generated code.

Reply:
No, this is not `@babel/core`. Please read the link I sent you. You are supposed to use it with `@babel/core` for backwards compatibility.

Reply:
It is, this is literally the entire code that package contains

```Javascript
module.exports = require("@babel/core");
```